### ISSUE TITLE
Creating a user - Add author / editor (role) field.

### ISSUE THREAD

**Issue by smypf** [MentallyFriendly/gbca#24]

**Comment by gillcosta** [comment]
@smypf What is the URL to add/remove users on the CMS?

**Comment by smypf** [comment]
@gillcosta http://gbca.destaging.com/gbca-admin/gbca/user/

This depends if you've got access though. It might be that you're not able to access the page.

If you log in using my account you'll be able to see this.

sasha@mentallyfriendly.com
sasha154

**Comment by gillcosta** [comment]
@smypf But the Editor should have full access right? Or do we have like a god mode user that has more access than the editor?

**Comment by smypf** [comment]
No. Editors are different. They have access to all the content in the system, but may or may not be able to manage users. This is reserved for superusers.

**Comment by gillcosta** [comment]
Ok cool. @MFCustServ Do we know who will be the super admin at their end?
